lib/memo_wise.rb

```diff
@@ -540,6 +540,7 @@ def reset_memo_wise(method_name = nil, *args, **kwargs)
       raise ArgumentError, "Provided args when method_name = nil" unless args.empty?
       raise ArgumentError, "Provided kwargs when method_name = nil" unless kwargs.empty?
 
+      # TODO: track and clear all memo_wise instance vars
```

Comment:
A few `reset_memo_wise` specs are failing due to the absence of this. Will add it when I figure out ^